kahuna/public/js/search/query.html

```diff
@@ -91,7 +91,7 @@
                            name="sort-direction"
                            ng-value="undefined"
```

Comment:
I appreciate this is legacy, but I think `"undefined"` should be replaced with "newest" or something throughout.

Reply:
I agree with you however, this is out of scope of this PR. I think it will be better if i implemented this in a separate PR.